tests/builders/test_rnn.py

```diff
@@ -49,44 +52,65 @@ def rnn_match_cfg(rnn: RNN, rnn_cfg: rnn_pb2.RNN, input_features: int) -> None:
         bias += getattr(rnn.rnn, f"bias_hh_l{l}")[
             hidden_size : 2 * hidden_size
         ]
-        assert torch.allclose(bias, torch.tensor(forget_gate_bias))
+        assert torch.allclose(
+            bias, torch.tensor(forget_gate_bias).to(bias.device)
+        )
 
 
 @st.composite
 def rnn_cfg_tensors(
     draw,
-) -> st.SearchStrategy[Tuple[torch.nn.Module, rnn_pb2.RNN, torch.Tensor]]:
+) -> st.SearchStrategy[
+    Tuple[torch.nn.Module, rnn_pb2.RNN, torch.Tensor, bool]
+]:
     """Returns a search strategy for RNNs built from a config + valid input."""
+    batch_first = draw(st.booleans())
     rnn_cfg = draw(rnns())
     tensor = draw(tensors(min_n_dims=3, max_n_dims=3))
-    rnn, _ = build(rnn_cfg, input_features=tensor.size(2))
-    return rnn, rnn_cfg, tensor
+    if batch_first:
+        tensor = tensor.transpose(1, 0)
+    rnn, _ = build(
+        rnn_cfg, input_features=tensor.size(2), batch_first=batch_first
+    )
+    return rnn, rnn_cfg, tensor, batch_first
 
 
 # Tests -----------------------------------------------------------------------
 
 
-@given(rnn_cfg=rnns(), input_features=st.integers(1, 128))
+@settings(deadline=3000)
```

Comment:
Can this deadline be reduced to reduce the iteration time for running the test suite? Currently this allows for 3s per run of the test of which there may be 100s.

Reply:
done - not sure why I added this as it runs within the `400ms` default.